helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1234,6 +1234,44 @@ certmanager:
   enabled: false
   installCRDs: true
 
+################################################################################
+# Target Allocator
+# Notice: Target Allocator related features should be considered to have an alpha
+# maturity level and be experimental. There may be breaking changes or Operator
+# features may be replaced entirely with a better alternative in the future.
+#
+# The Target Allocator is running as a deployment with a replica count of 1.
+# It discovers scraping configurations from ServiceMonitor and PodMonitor CRDs and
+# assigns them to collectors.
+# Related documentation: https://github.com/open-telemetry/opentelemetry-operator/tree/main/cmd/otel-allocator
+################################################################################
+
+targetallocator:
```

Comment:
Maybe `targetAllocator` to follow helm best practices?

Reply:
renamed